tutorial-review.md

```diff
@@ -0,0 +1,1053 @@
+---
+title: "Deploy a MERN stack with Express, React and MongoDB Atlas serverless database"
+description: "This first part of MERN series shows how to build a Blog application and deploy it on Koyeb Serverless Platform."
+tags: ["MongoDB", "Node", "Express", "React", "MERN"]
+editUrl: "pages/tutorials/deploy-a-mern-stack-with-express-react-and-mongodb-atlas-serverless-database"
+lastEdited: "2022-03-15T00:00:00.000Z"
+authors:
+  - fullname: "Samuele Zaza"
+    avatar: "/static/images/team/szaza.jpg"
+---
+
+## Introduction
+
+This is the first article of a mini-series focused on the popular MERN (Mongo, Express, React, Node) stack.
+We will create a Blog sample app as it gives us a chance to dig into the MERN stack, building the fundamental blocks and connect the dots. At the end we should have a full-functioning basic Blog web app where authors can post, edit and delete articles. To complete the tutorial, the app will be deployed on the internet by using Koyeb serverless platform.
+The second part, instead, will focus on creating a microservice to add extra search capabilities to the blog app by using mongo search atlas.
+
+## Requirements
+
+- A local environment with NPM and Node.js installed
+- Basic knowledge of JavaScript, Express, React and querying a MongoDB database using Mongoose
+- A [GitHub account](https://github.com) to version and deploy your application code on Koyeb
+- A [Koyeb account](https://app.koyeb.com) to deploy and run the application
+
+## Steps
+
+1. Set up the project
+2. Create a MongoDB Atlas database
+3.
+4.
+
+## Set up the project
+
+### Create the project client and server repos
+
+Let's kick off the project by creating the repo `mongo-blog` and installing all the related dependencies.
+
+Open your terminal and create the project folder:
+```bash
+mkdir mongo-blog
+```
+
+Move into `mongo-blog` first and setup Express using `express-generator`:
+
+```bash
+cd mongo-blog
+npx express-generator
```

Comment:
Could you add an explanation of what is npx and why you're using it in this project?